src/relay/handshake.ts

```diff
@@ -269,7 +270,7 @@ class RelayHandshake {
         this.shaker.rest()
         destinationShaker.rest()
 
-        createNew(source, destination, this.shaker.stream, destinationShaker.stream)
+        createNew(source, destination, this.shaker.stream, destinationShaker.stream, relayFreeTimeout)
```

Comment:
why don't prefix with `__` to make sure that it is considered a variable that is only used for testing